### PR TITLE
feat(Websim): add presence

### PR DESCRIPTION
**Pull request by gwashark**
## Description 

Initial commit for a new presence

## Acknowledgements
- [x] I read the [Presence Guidelines](https://github.com/PreMiD/Presences/blob/main/.github/CONTRIBUTING.md)
- [x] I linted the code by running `yarn format`
- [x] The PR title follows the repo's [commit conventions](https://github.com/PreMiD/Presences/blob/main/.github/COMMIT_CONVENTION.md)

## Screenshots
image
image
image

</details>

### CODE REVIEW

File: websites/W/Websim/presence.ts [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+	}),
+	browsingTimestamp = Math.floor(Date.now() / 1000);
+
+presence.on("UpdateData", async () => {
+	const showButtons = await presence.getSetting("showButtons"),
+		btnPrivacy = await presence.getSetting("btnPrivacy"),
+		presenceData: PresenceData = {
+			startTimestamp: browsingTimestamp,
+		},
+		path = window.location.pathname;
```
Comment (theusaf) [inline websites/W/Websim/presence.ts:12]:
```suggestion
		path = document.location.pathname;
```

File: websites/W/Websim/presence.ts [hunk 2]
```diff
@@ -0,0 +1,71 @@
+const presence = new Presence({
+		clientId: "1315383978878046411",
+	}),
+	browsingTimestamp = Math.floor(Date.now() / 1000);
+
+presence.on("UpdateData", async () => {
+	const showButtons = await presence.getSetting("showButtons"),
+		btnPrivacy = await presence.getSetting("btnPrivacy"),
```
Comment (theusaf) [inline websites/W/Websim/presence.ts:8]: Combine these using `Promise.all`

File: websites/W/Websim/presence.ts [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+		presenceData: PresenceData = {
+			startTimestamp: browsingTimestamp,
```
Comment (theusaf) [inline websites/W/Websim/presence.ts:10]:
```suggestion
			startTimestamp: browsingTimestamp,
			largeImageKey: "https://i.imgur.com/zDNmRxT.png",
```

**Review by theusaf — CHANGES_REQUESTED** [review]

**Comment by gwashark** [comment]
@theusaf there wasn't anything to merge with...

**Pull request merged**